### ISSUE TITLE
Question about zlib

### ISSUE THREAD

**Issue by kalwalt** [Carnaux/ZFT-Creator#1]
Since Emscripten add is zlib see https://github.com/Carnaux/ZFT-Creator/blob/016b60d8ad256ca65f70896157cabbb5655e6b0b/tools/makem.js#L62

I'm asking if the version you added is different by the Emscripten zlib.

**Comment by Carnaux** [comment]
No, emscripten ports the 1.2.8 version, and the one I added is the 1.2.11. I used the flag to fix the undefined symbols.

**Comment by kalwalt** [comment]
Ok understood, so it is necessary because a different version. i tested compressing and decompressing and it works very good. Thinking for the other steps...
